### PR TITLE
Simplify KubeVirt API types

### PR DESCRIPTION
**Pull request by hdurand0710**
Signed-off-by: Helene Durand <helene@kubermatic.com>

**What does this PR do / Why do we need it**:
Goal is to simplify the api types for Kubevirt (flavor, storage class). For Flavor, we keep only as properties the name, namespace and a marshalled string of the Flavour Spec. For the Storage Class, only the name.

**Does this PR close any issues?**:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/api/v2/types.go [hunk 1]
```diff
@@ -1062,55 +1059,13 @@ type GKEUpgradeSettings struct {
...
-}
-
 // VirtualMachineInstancePreset represents a KubeVirt Virtual Machine Instance Preset
 // swagger:model VirtualMachineInstancePreset
 type VirtualMachineInstancePreset struct {
-	apiv1.ObjectMeta `json:",inline"`
-	// VirtualMachineInstance Spec contains the VirtualMachineInstance specification.
-	Spec VirtualMachineInstancePresetSpec `json:"spec,omitempty" valid:"required"`
+	Name      string `json:"name,omitempty"`
+	Namespace string `json:"namespace,omitempty"`
```
Comment (maciaszczykm) [inline pkg/api/v2/types.go:1066]: Is the namespace required, will it be displayed in the UI?

Comment (hdurand0710) [reply pkg/api/v2/types.go:1066]: That's a good point. When implementing [Initialize Virtual Machine Presets](https://github.com/kubermatic/kubermatic/issues/8490), we will fetch Presets from the `default` namespace, but also add a/several `standard` hard-coded Presets (empty namespace). I think that it's good to display then the namespace (`default` or empty).

Comment (maciaszczykm) [reply pkg/api/v2/types.go:1066]: Okay 👍🏻

**Pull request merged**